### PR TITLE
Bill/header updates

### PR DESCRIPTION
**Pull request by bill-luu**
### Changelog

Removed the header on each of the pages, leaving just the alive button on it. Also applied scaling for multiple devices and changed some text colors.

### CODE REVIEW

File: web-app/src/app/tabs/battery/battery.component.css [hunk 1]
```diff
@@ -1,65 +1,45 @@
...
-	height: 15px;
+	width: 1em;
+	height: 1em;
 }
 .circleAlive {
 	float: left;
-	width: 15px;
-	height: 15px;
+	width: 1em;
+	height: 1em;
```
Comment (henrytran28) [inline web-app/src/app/tabs/battery/battery.component.css]: Just curious what the difference between circleAlive and circle is? They have the same css but one has margin-top 2px. The Battery page uses .circle and the Controls page uses the circleAlive. The circle shifts by 2px lower than the text on controls.

File: web-app/src/app/tabs/battery/battery.component.css [hunk 2]
```diff
@@ -1,65 +1,45 @@
...
+	margin-bottom: 0.1em;
+	width: auto;
 }
-.title-left {
-	float: left;
-	width: 50%;
-	margin-left: 5%;
+.aliveGroup {
+	margin-left: auto;
+	width: auto;
```
Comment (henrytran28) [inline web-app/src/app/tabs/battery/battery.component.css]: Don't think this line changes anything.

File: web-app/src/app/tabs/battery/battery.component.css [hunk 3]
```diff
@@ -1,65 +1,45 @@
...
+	padding-bottom: 0.2em;
 	padding-top: .2em;
-	border-radius: 25px;
-	margin-bottom: 25px;
-	margin-top: 10px;
+	margin-top: 0.1em;
+	margin-bottom: 0.1em;
+	width: auto;
 }
-.title-left {
```
[the remark this answers is not in the document]
Comment (bill-luu) [reply web-app/src/app/tabs/battery/battery.component.css]: Done in the Battery PR I think

File: web-app/src/app/tabs/battery/battery.component.css (hunk 1 above)
Comment (bill-luu) [reply web-app/src/app/tabs/battery/battery.component.css]: I don't think it (intentionally) does anything different, I'll remove it and apply those changes to Battery

**Comment by bill-luu** [comment]
idk how I accidentally merged this.... responding to PR comments in #17

**Pull request merged**